approved/v93k_smt8/OrigenTesters/flows/FLOW_CONTROL.flow

```diff
@@ -1751,5 +1765,15 @@ flow FLOW_CONTROL {
         else
         {
         }
+        for @index = 0; @index < @loop ; @index = @index + 1; do
```

Comment:
I doubt this is valid smt8, its fine to not support it on SMT8 yet, but in that case either output nothing and emit a warning, or else raise an error rather than generating garbage.

Reply:
@info-rchitect  can you look at this? adding it in a tester.smt7? check should be okay